docs/tutorials/hospital_admissions_model.qmd

```diff
@@ -1,41 +1,51 @@
 ---
-title: Fitting a hospital admissions-only model
+title: "Fitting a hospital admissions-only model"
 format: gfm
 engine: jupyter
 ---
 
-This document illustrates how a hospital admissions-only model can be fitted using data from the Pyrenew package, particularly the wastewater dataset. The CFA wastewater team created this dataset, which contains simulated data.
+This document illustrates how a hospital admissions-only model can be fitted using simulated example data from the `pyrenew` package which was created by the CFA wastewater team.
```

Comment:
```suggestion
This document illustrates how a hospital admissions-only model can be fitted using simulated example data.
```

I know what this is trying to say, and it's not worth mentioning. The data is simulated from an old stan model we used to use.